index.html

```diff
@@ -2,23 +2,23 @@
 <html>
 <head>
     <!-- Standard Meta Tags-->
-    <title>BrickHack 7</title>
+    <title>BrickHack Club</title>
     <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1 minimum-scale=1">
     <meta name="keywords" content="hackathons, hackathon, hack-a-thon, programming, hardware, development, design, rit"/>
     <meta name="description" content="RIT's premiere collegiate hackathon devoted to bringing designers, developers, and all sorts of makers together for 24 hours to create something from nothing.">
 
     <!-- Twitter Cards -->
     <meta property="twitter:card" content="summary"/>
-    <meta property="twitter:title" content="BrickHack 7">
+    <meta property="twitter:title" content="BrickHack 8">
     <meta property="twitter:url" content="https://brickhack.io" >
     <meta property="twitter:description" content="RIT's premiere collegiate hackathon devoted to bringing designers, developers, and all sorts of makers together for 24 hours to create something from nothing.">
     <meta name="twitter:image" content="./assets/social_cards.png">
```

Comment:
Are we still using the BH7 card image for the BH8 preregister site, or are we making a new one? IMO this image doesn't fit the new theming.
![image](https://user-images.githubusercontent.com/16471311/123360472-ffb40100-d532-11eb-87d1-9d01af4962d9.png)

Reply:
It does not and yes I can swap it for a screenshot of our header logo on that background color